### ISSUE TITLE
Maui Blazor No Hot Reload.

### ISSUE THREAD

**Issue by cabbytech** [dotnet/maui#10367]
### Description

If you create a new Maui Blazor app in .NET 6 Hot Reload is showing as unavailable. 
Microsoft Visual Studio Community 2022
Version 17.4.0 Preview 2.1
VisualStudio.17.Preview/17.4.0-pre.2.1+32916.344

### Steps to Reproduce

1. Create a new app in VS 2022 .NET 6
2. Run the app.
3. Hot Reload is not available.

### Link to public reproduction project repository

https://github.com/dotnet/maui

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 11

### Did you find any workaround?

No

### Relevant log output

```shell
None
```

**Comment by cabbytech** [comment]
On the .NET 6 Maui Blazor template when you run the app on a Windows machine without any changes there is a set of icons at the top of the index page to help debugging. The icon relating to  Hot Reload icon states that Hot Reload is not available. Presumably razor or CSS.

**Comment by cabbytech** [comment]
Hi @TanayParikh. Yes the that was what I was referring to. I did have Hot-Reload enabled but it was still not available. I was going to take a snip of the screen to show you but when I reloaded the app it was working. It must have been my machine. However, there was a Stackoverflow  member that was having the same problem. https://stackoverflow.com/questions/73853721/maui-blazor-no-hot-reload/73861455?noredirect=1#comment130445430_73861455
